package.json

```diff
@@ -77,7 +77,7 @@
     "prettier": "2.7.1",
     "prompts": "^2.4.2",
     "rimraf": "^3.0.2",
-    "rollup": ">=2.75.6 <2.77.0 || ~2.77.0",
+    "rollup": "^2.78.0",
```

Comment:
Maybe we should continue to fix the rollup version to minors? Using `~2.78.0`, or maybe `>=2.78.0 <2.79.0` and bump the upper bound as we see each Rollup minor is safe for Vite users?

Reply:
Update to ~ for now